collection.go

```diff
@@ -45,7 +45,11 @@ func (c Collection) Encode() []byte {
 
 // A CollectionGuarantee is an attestation signed by the nodes that have guaranteed a collection.
 type CollectionGuarantee struct {
-	CollectionID Identifier
+	CollectionID     Identifier
+	Signatures       [][]byte
```

Comment:
you can skip this field. it's legacy from before we started using aggregate signatures, and is hardcoded to return the same value as `Signature`
https://github.com/onflow/flow-go/blob/ffaa335bacd8d5108ddf9a089278268e6ba10674/engine/common/rpc/convert/collections.go#L93